Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN sh -c "echo 'Package: *\nPin: release o=repo.radeon.com\nPin-priority: 600'
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     apt-utils \
     build-essential \
+    clang-format \
```

Comment:
It should not be using this clang-format. It should be using the one in /opt/rocm/llvm/bin/clang-format.